### PR TITLE
add odenetpkg

### PR DESCRIPTION
**Pull request by uqyge**
Checklist

- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vend other packages
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your
      recipe (see
      [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos)
      for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

File: recipes/ODENet/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+  license: MIT
+  license_family: MIT
+  license_file: LICENSE
+  summary: 'ODENet for combustion'
+
+extra:
+  recipe-maintainers:
+    - uqyge
+    - icehuli
+    - mhansinger
```
Comment (marcelotrevisani) [inline recipes/ODENet/meta.yaml]: @icehuli and @mhansinger
Could you please confirm that you want to maintain this recipe as well?

File: recipes/ODENet/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+  name: {{ name|lower }}
+  version: {{ version }} 
+
+source:
+  url: https://github.com/uqyge/{{ name }}/archive/{{ version }}.tar.gz
+  sha256: d060a64004b12fbb61eff60618f8afd5c843f4dc4f5289b5df195269e69a23ab
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
```
Comment (marcelotrevisani) [inline recipes/ODENet/meta.yaml]: I think this recipe can be ``noarch``
```suggestion
  script: "{{ PYTHON }} -m pip install . -vv"
  noarch: python
```

File: recipes/ODENet/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+
+requirements:
+  host:
+    - python
+    - pip
+  run:
+    - python
+    - tensorflow
```
Comment (marcelotrevisani) [inline recipes/ODENet/meta.yaml]: It seems that ``pandas`` is a dependency. Please add:
```suggestion
    - tensorflow
    - pandas
```

File: recipes/ODENet/meta.yaml (hunk 2 above)
Comment (uqyge) [reply recipes/ODENet/meta.yaml]: thx so much, I am committing the suggestion. It's my first time doing conda packages, could you point me to some material about the reason for doing noarch?

File: recipes/ODENet/meta.yaml (hunk 1 above)
Comment (uqyge) [reply recipes/ODENet/meta.yaml]: It' holiday today in Germany. I will contact my collages and they will confirm their roles by tmr.

File: recipes/ODENet/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+  noarch: python
+
+requirements:
+  host:
+    - python
+    - pip
+  run:
+    - python
+    - tensorflow
+    - pandas
```
Comment (marcelotrevisani) [inline recipes/ODENet/meta.yaml]: You will also need to add ``scikit-learn``
```suggestion
    - pandas
    - scikit-learn
```

**Pull request merged**